Vagrantfile.example

```diff
@@ -0,0 +1,61 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+ config.vm.box = "fedora/25-cloud-base"
```

Comment:
I recommend using the URL that gets it from Fedora's mirror network instead of this, which gets it from Hashicorp. It's unsigned either way, so it's arguable as to whether what I'm saying is really better. Here's the URL if you are interested:

https://download.fedoraproject.org/pub/fedora/linux/releases/25/CloudImages/x86_64/images/Fedora-Cloud-Base-Vagrant-25-1.3.x86_64.vagrant-libvirt.box